pfcpiface/bess.go

```diff
@@ -403,7 +428,7 @@ func (b *bess) sim(u *upf, method string) {
 			fseID:     uint64(n3TEID + i),
 			ctrID:     i,
 			farID:     n3,
-			qerID:     n3,
+			qerID:     n3 + 5,
```

Comment:
can we remove this + 5?

Reply:
Lets have 2 QER - 1 default UL/DL MBR and other UL/DL GBR.

PDR N3->N9 and N9->N3 will get the default QER ID
PDR N3->N6 and N6->N3 will get the other QER ID